reference/6/Microsoft.PowerShell.Core/About/about_CommonParameters.md

```diff
@@ -8,29 +8,29 @@ title:  about_CommonParameters
 # About CommonParameters
 
 ## SHORT DESCRIPTION
+
 Describes the parameters that can be used with any cmdlet.
 
 ## LONG DESCRIPTION
 
 The common parameters are a set of cmdlet parameters that you can use with any
-cmdlet. They are implemented by PowerShell, not by the cmdlet
-developer, and they are automatically available to any cmdlet.
+cmdlet. They're implemented by PowerShell, not by the cmdlet developer, and
+they're automatically available to any cmdlet.
 
 You can use the common parameters with any cmdlet, but they might not have an
-effect on all cmdlets. For example, if a cmdlet does not generate any verbose
-output, using the Verbose common parameter has no effect.
+effect on all cmdlets. For example, if a cmdlet doesn't generate any verbose
+output, using the `-Verbose` common parameter has no effect.
```

Comment:
Only use the dashed form of the parameter when you are showing it in an example. Just use the parameter name and make it bold (not fenced).

Reply:
Fixed.